tests/bluetooth/bt_crypto/bt-crypto-mbedtls.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
+CONFIG_MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG=y
```

Comment:
Is this one mandatory?

Reply:
RNG is mandatory to support for bluetooth pairing and connection (link) establishment and probably a lot of other use-cases.

Reply:
Yes, but my question is why `EXTERNAL` and not `LEGACY`?

Reply:
The idea was to keep the PSA crypto implementation to a minimum. Using legacy means add few other modules to Mbed TLS like entropy and ctr/hmac-drbg. I can give this a try though.

Reply:
Yeah, though by using `EXTERNAL` you add the dependency on `CONFIG_CSPRNG_ENABLED` in the test, so I don't know whether it will actually ever be run in CI? From that point of view, it could be nice if the test could run on `native_sim`.

Reply:
And here it's a test so I don't think we care much about code size as long as it doesn't overflow... Unless you want to measure the difference between the TC and PSA variants using the unmodified test, obviously.

Reply:
> hough by using `EXTERNAL` you add the dependency on `CONFIG_CSPRNG_ENABLED` in the test, so I don't know whether it will actually ever be run in CI?

Just for the record, testing just the mbedtls case on _all_ platform with `twister -c -s tests/bluetooth/bt_crypto/bluetooth.bt_crypto.mbedtls --all` makes the test built and _executed_ on 4 of them:
```
INFO    - 4 of 733 test configurations passed (100.00%), 0 failed, 0 errored, 729 skipped with 0 warnings in 12.16 seconds
[...]
INFO    - 4 test configurations executed on platforms, 0 test configurations were only built.
```

but I can try legacy RNG to see if this way I can make it tested on more platforms.

Reply:
I changed to legacy RNG in https://github.com/zephyrproject-rtos/zephyr/pull/73378/commits/4ff681715c66aeb93caaf411d8415c1bc2c21c0f. However I still had to add `CONFIG_ENTROPY_HAS_DRIVER` filter on Mbed TLS because this is needed to use some valid random generator (ex: `ENTROPY_DEVICE_RANDOM_GENERATOR` or `XOSHIRO_RANDOM_GENERATOR` and not `TIMER_RANDOM_GENERATOR`).
In this way the mbedtls test is still run on 4 platforms out of 733, so it seems that this change did not extend "testing coverage" to include more platforms. Let me know if you prefer this new option or the previous one, please.

Reply:
Okay, thanks for testing out. I don't have a preference.